spec/control_classes/tcp_server.rb

```diff
@@ -0,0 +1,7 @@
+class TCPServer
```

Comment:
Probably not going to work exactly (unless we use something other than socket), because the TCPServer object itself is call this and it would shadow that naming.